applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_explicit_dynamic_solver.py

```diff
@@ -0,0 +1,89 @@
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import structural_mechanics_solver
+
+# Check that KratosMultiphysics was imported in the main script
+KratosMultiphysics.CheckForPreviousImport()
+
+
+def CreateSolver(main_model_part, custom_settings):
+    return ExplicitMechanicalSolver(main_model_part, custom_settings)
+
+class ExplicitMechanicalSolver(structural_mechanics_solver.MechanicalSolver):
+    """The structural mechanics explicit dynamic solver.
+
+    This class creates the mechanical solvers for explicit dynamic analysis.
+
+    Public member variables:
+    dynamic_settings -- settings for the explicit dynamic solvers.
+
+    See structural_mechanics_solver.py for more information.
+    """
+    def __init__(self, main_model_part, custom_settings):
+         # Set defaults and validate custom settings.
+        self.dynamic_settings = KratosMultiphysics.Parameters("""
+        {
+            "scheme_type" : "central_differences",
+            "time_step_prediction_level": 0,
+            "max_delta_time": 1.0e-5,
+            "fraction_delta_time": 0.9
+        }
+        """)
+        self.validate_and_transfer_matching_settings(custom_settings, self.dynamic_settings)
+        # Validate the remaining settings in the base class.
+
+
+        # Construct the base solver.
+        super(ExplicitMechanicalSolver, self).__init__(main_model_part, custom_settings)
+        self.print_on_rank_zero("::[ExplicitMechanicalSolver]:: Construction finished")
+
+    def AddVariables(self):
+        super(ExplicitMechanicalSolver, self).AddVariables()
+        self._add_dynamic_variables()
+        self.main_model_part.AddNodalSolutionStepVariable(StructuralMechanicsApplication.MIDDLE_VELOCITY)
```

Comment:
to be discussed, if for this you really need the historical database (based on the comment of @loumalouomega )